packages/serverless-orchestration/test/ServerlessHub.js

```diff
@@ -124,11 +124,10 @@ contract("ServerlessHub.js", function(accounts) {
 
     uniswap = await UniswapMock.new();
 
-    defaultUniswapPricefeedConfig = {
```

Comment:
Removing reliance on UniswapMock because this package uses @uma/core version 1.2 which doesn't have latest changes to UniswapMock.sol